packages/expo-notifications/src/scheduleNotificationAsync.ts

```diff
@@ -29,14 +29,42 @@ function parseTrigger(userFacingTrigger: NotificationTriggerInput): NativeNotifi
     return { type: 'date', timestamp: userFacingTrigger.getTime() };
   } else if (typeof userFacingTrigger === 'number') {
     return { type: 'date', timestamp: userFacingTrigger };
-  } else if ('seconds' in userFacingTrigger) {
+  } else if (
+    Object.keys(userFacingTrigger).length === 2 &&
+    'hour' in userFacingTrigger &&
+    'minute' in userFacingTrigger
+  ) {
+    const hour = userFacingTrigger.hour;
+    const minute = userFacingTrigger.minute;
+    if (hour === undefined || minute === undefined) {
+      throw new TypeError('Both hour and minute need to have valid values. Found undefined');
+    }
+    if (hour < 0 || hour > 23) {
+      throw new RangeError(`The hour parameter needs to be between 0 and 23. Found: ${hour}`);
+    }
+    if (minute < 0 || minute > 59) {
+      throw new RangeError(`The minute parameter needs to be between 0 and 59. Found: ${minute}`);
+    }
+    return {
+      type: 'daily',
+      hour,
+      minute,
+    };
+  } else if (
+    (Object.keys(userFacingTrigger).length === 1 && 'seconds' in userFacingTrigger) ||
+    (Object.keys(userFacingTrigger).length === 2 &&
+      'seconds' in userFacingTrigger &&
+      'repeats' in userFacingTrigger)
+  ) {
     return {
       type: 'timeInterval',
       seconds: userFacingTrigger.seconds,
       repeats: userFacingTrigger.repeats ?? false,
     };
   } else {
+    // @ts-ignore
     const { repeats, ...calendarTrigger } = userFacingTrigger;
+    // @ts-ignore
     return { type: 'calendar', value: calendarTrigger, repeats };
```

Comment:
TS lint noted that DailyTrigger for example doesn't have `repeats`. I could rewrite this so that it only sets it conditionally but opted for ts-ignore instead. Let me know what you think about it.

Reply:
Yeah, I think ts-ignoring it makes sense for now 👍 (unless covering all other `seconds` cases earlier makes TS infer that down here the only possibility is for the `userFacingTrigger` to be an instance of `CalendarTriggerInput` which is the case now).